library/oraclelinux

```diff
@@ -1,39 +1,57 @@
 Maintainers: Oracle Linux Product Team <ol-ovm-info_ww@oracle.com> (@Oracle)
-GitRepo: https://github.com/oracle/ol-container-images.git
+GitRepo: https://github.com/oracle/container-images.git
+GitCommit: 01a15ec99c7470a3391c691509db1759b41eaf66
 GitFetch: refs/heads/master
-GitCommit: 5a8efb148fb726e64e03252e44af87ae993bf483
+
```

Comment:
This empty line is telling `bashbrew` that the following block of headers is a new entry, and thus needs `Tags:`.  Removing it (and the one following this block before `Constraints:` which also needs to be part of the global section of the file) should DTRT.